teaching-platform/models.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+
+from extensions import db
+
+
+class User(db.Model):
+    __tablename__ = "users"
+    id = db.Column(db.Integer, primary_key = True)
+    username = db.Column(db.String(20), unique=True, nullable= False)
+    password = db.Column(db.String(90),nullable= False)
```

Comment:
90 is kind of arbitrary - you are storing hashes, i dont think that there is a reason to specify max length of password hash entry.

Reply:
done.